src/duration.rs

```diff
@@ -232,6 +232,20 @@ impl Duration {
         }
     }
 
+    /// Returns the number of nanoseconds such that
+    /// `subsec_micros() + num_seconds() * MICROS_PER_SEC` is the total number of
```

Comment:
> Returns the number of *nano*seconds ...

I would have guess the function name `subsec_micros` alluded to *micro*seconds.  This seems confusing to me.